### PR TITLE
Fix attachment icon in print bug page

### PR DESCRIPTION
**Pull request by syncguru**
Fixes #21139

screen shot 2016-08-14 at 5 24 08 pm

### CODE REVIEW

File: print_bug_page.php [hunk 1]
```diff
@@ -529,7 +529,9 @@
 		$c_filesize = number_format( $t_attachment['size'] );
 		$c_date_added = date( config_get( 'normal_date_format' ), $t_attachment['date_added'] );
 		if( isset( $t_attachment['icon'] ) ) {
-			echo '<img src="', $t_attachment['icon']['url'], '" alt="', $t_attachment['icon']['alt'], '" />&#160;';
+			echo '<i class="ace-icon fa fa-lg ', $t_attachment['icon']['url'], '" alt="', $t_attachment['icon']['alt'], '"></i>&#160;';
```
Comment (vboctor) [inline print_bug_page.php]: Does it make sense to replace ['icon']['url'] with something more meaningful? e.g. ['icon']['name']?

**Comment by vboctor** [comment]
:+1:

**Comment by atrol** [comment]
:+1:

**Comment by dregad** [comment]
Merged in d7aa38b7de38bb7d3f90306d4d89561e973c68c5

**Pull request closed without merging**